subsys/net/lib/nrf_cloud/src/nrf_cloud_rest.c

```diff
@@ -343,8 +343,6 @@ static int do_api_call(struct http_request *http_req, struct nrf_cloud_rest_cont
 		err = 0;
 	}
 
-	close_connection(rest_ctx);
```

Comment:
When will it be closed?

Reply:
each function calls `close_connection` before returning